src/components/VerticalCard/VerticalCard.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useCart } from 'context/CartContext'
+import { decreaseCartQuantity, deleteCart, IncreaseCartQuantity, addItemInWishlist } from 'utilities/helpers/http-helper';
+import { useAuth } from "context/AuthenticationContext";
+import "./VerticalCard.css";
+
+const VerticalCard = ({ item }) => {
+    const { token } = useAuth();
+    const data = item;
+    const { cartDispatch, wishlistDispatch, wishlistItem } = useCart();
+    const moveToWishlistHandler = () => {
+        if (wishlistItem.find(item => item.id === data.id)) {
+            return;
+        } else {
+            wishlistDispatch({ type: "ADD_TO_WISHLIST", payload: data })
+            addItemInWishlist(data, token)
+        }
+        cartDispatch({ type: "REMOVE_FROM_CART", payload: data })
+        deleteCart(data, token)
+    }
+    const deleteCartHandler = () => {
+        cartDispatch({ type: "REMOVE_FROM_CART", payload: data })
+        deleteCart(data, token)
+    }
+    const increaseQuantityHandler = () => {
+        cartDispatch({ type: "INCREASE_CART_QUANTITY", payload: data })
+        IncreaseCartQuantity(data, token)
+    }
+
+    const decreaseQuantityHandler = () => {
+        cartDispatch({ type: "DECREASE_CART_QUANTITY", payload: data })
+        decreaseCartQuantity(data, token)
+    }
+    return (
+        <>
+            <div className="card-horizontal">
+                <div className="image-container">
+                    <img src={data.image} alt="card_1" className="card-image" />
+                </div>
+                <div className="text-container">
+                    <div className="text-container-title">
+                        <h6>{data.title}</h6>
+                    </div>
+                    <div className="text-container-desc">
+                        <p>by {data.author}</p>
```

Comment:
You can further de-structure it, so as to avoid repetition of the same word. For eg: instead of `src={data.image}` you can use `src={image}` after destructuring your data.